packages/service-library/src/servicelib/rabbitmq/_client.py

```diff
@@ -20,6 +21,63 @@
 
 _DEFAULT_PREFETCH_VALUE: Final[int] = 10
 _DEFAULT_RABBITMQ_EXECUTION_TIMEOUT_S: Final[int] = 5
+_HEADER_X_DEATH: Final[str] = "x-death"
+
+DEFAULT_UNEXPECTED_ERROR_RETRY_DELAY_S: Final[float] = 1
+DEFAULT_UNEXPECTED_ERROR_MAX_ATTEMPTS: Final[NonNegativeInt] = 15
+
+
+def _get_delayed_exchange_name(exchange_name: str) -> str:
+    return f"delayed_{exchange_name}"
```

Comment:
THOUGHT: This is a recursive comment. IMO for `str`formatting you do not need to do function calls. It obscures the interface by having functions for things that can be done in place with a known interface.

```python

assert _get_delayed_exchange_name(exchange_name="foo") == DELAYED_EXHANGE_NAME.format(exchange_name="foo")
```
The first suggests you are doing some call somewhere or some sophisticated operation to get a name, the second is tells you that is a simple string formatting

Reply:
This is actually nicer. Thanks for the suggestion.